tests/mock_tables/appl_db.json

```diff
@@ -532,8 +532,7 @@
     "speed": 100000
   },
   "PORT_TABLE:Ethernet116": {
-    "description": "snowflake",
-    "speed": 100000
+    "speed": 1000
```

Comment:
>speed": 1000 [](start = 5, length = 12)

Seems you don't need to change speed in this PR.

Reply:
agreed it's a relic of the other PR, if we leave it it will be one less conflict at merge